### PR TITLE
fix/BODS-400 + BODS-401 + BODS-402: Fares Validator mandatory check bug fixes

### PR DESCRIPTION
**Pull request by ndemenezes**

### CODE REVIEW

File: transit_odp/fares_validator/tests/test_mandatory_type_of_frame_refs.py [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+        (
+            True,
+            True,
+            True,
+            True,
+            False,
+            [
+                "violation",
+                "8",
+                "Attribute 'ref' of element 'TypeOfFrameRef' does not contain 'UK_PI_FARE_PRICE'",
```
Comment (surabhika) [inline transit_odp/fares_validator/tests/test_mandatory_type_of_frame_refs.py:55]: [1]:This error message does not match one defined in validation_message.py. There is a small difference in wording.

File: transit_odp/fares_validator/tests/test_mandatory_type_of_frame_refs.py [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+        (
+            True,
+            True,
+            True,
+            True,
+            False,
+            [
+                "violation",
+                "8",
+                "Attribute 'ref' of element 'TypeOfFrameRef' does not contain 'UK_PI_FARE_PRODUCT'",
```
Comment (surabhika) [inline transit_odp/fares_validator/tests/test_mandatory_type_of_frame_refs.py:170]: see [1]

File: transit_odp/fares_validator/tests/test_mandatory_type_of_frame_refs.py [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+        (
+            True,
+            True,
+            True,
+            True,
+            False,
+            [
+                "violation",
+                "8",
+                "Attribute 'ref' of element 'TypeOfFrameRef' does not contain 'UK_PI_COMMON'",
```
Comment (surabhika) [inline transit_odp/fares_validator/tests/test_mandatory_type_of_frame_refs.py:296]: see [1]

Comment (ndemenezes) [reply transit_odp/fares_validator/tests/test_mandatory_type_of_frame_refs.py]: Fixed the wording on all

**Review by surabhika — APPROVED** [review]

**Pull request merged**